istioctl/pkg/writer/envoy/configdump/configdump.go

```diff
@@ -53,6 +85,9 @@ func (c *ConfigWriter) PrintBootstrapDump() error {
 	if err != nil {
 		return err
 	}
+	istioProxyVersion, envoyVersion := getVersionInfo(bootstrapDump)
+	fmt.Println("Istio Proxy Version: ", istioProxyVersion)
```

Comment:
We don't want to output the JSON and the human-readable text together.

If we do this, it won't be possible to pipe the JSON into a tool like `jq`, and the human readable text will scroll away.

Look at the `--output` flag like the other subcommands, and follow what they do.

Reply:
summaryOutput is the default for proxyConfig commands
https://github.com/istio/istio/blob/master/istioctl/cmd/proxyconfig.go#L663

```➜  istio git:(GH-21517) <minikube>
$ ./out/darwin_amd64/release/istioctl-osx proxy-config bootstrap prometheus-5fdfc44fb7-q96bm.istio-system --output=short 
Istio Proxy Version:  1.6.3(istio-proxy:9085616b434c0e0b54cc677cfd3b97a0fed71c0c)
Envoy Version:        1.14-1/Clean/RELEASE/BoringSSL
➜  istio git:(GH-21517) <minikube>
$ ./out/darwin_amd64/release/istioctl-osx proxy-config bootstrap prometheus-5fdfc44fb7-q96bm.istio-system                
Istio Proxy Version:  1.6.3(istio-proxy:9085616b434c0e0b54cc677cfd3b97a0fed71c0c)
Envoy Version:        1.14-1/Clean/RELEASE/BoringSSL
```


Users will have to explicitly specify `./out/darwin_amd64/release/istioctl-osx proxy-config bootstrap prometheus-5fdfc44fb7-q96bm.istio-system --output=json` to get JSON configdump. 

Checking how to make this backward compatible else it can break scripts for users that rely on bootstrap command output.

Reply:
@esnible should I just a `--version` boolean flag instead of using `--output`?